ml-agents/mlagents/trainers/learn.py

```diff
@@ -170,7 +169,7 @@ def parse_command_line(argv: Optional[List[str]] = None) -> CommandLineOptions:
         "--cpu", default=False, action="store_true", help="Run with CPU only"
     )
 
-    parser.add_argument("--version", action="version", version=get_version_string())
+    parser.add_argument("--version", action="version", version="")
```

Comment:
`--version` still works, but we always print the version anyway. I think it's worth keeping since that's a standard argument for a CLI.